dhis-2/dhis-api/src/test/java/org/hisp/dhis/category/CategoryOptionComboTest.java

```diff
@@ -145,7 +145,7 @@ void hasDefault() {
 
   @Test
   void testIsDefault() {
-    categoryComboA.setName(CategoryCombo.DEFAULT_CATEGORY_COMBO_NAME);
+    optionComboA.setName(CategoryCombo.DEFAULT_CATEGORY_COMBO_NAME);
```

Comment:
Likewise, I suggest `optionComboA.setName(CategoryOptionCombo.DEFAULT_NAME)`. You're setting a CatOptionCombo name so it's better form to use the CatOptionCombo default name.